mu/modes/pyboard.py

```diff
@@ -31,7 +31,7 @@ class PyboardMode(MicroPythonMode):
 
     name = _("Pyboard MicroPython")
     short_name = "pyboard"
-    description = _("Use MicroPython on the Pyboard line of boards")
+    description = _("Use MicroPython on the Pyboard line of boards.")
```

Comment:
We cannot change this file on this PR: otherwise the work from #1872 becomes stale, given that it includes references (msgids?) to the unchanged text! ;-)

Reply:
Fixed in collaboration with @xbecas. Cool!